root/curs/front_page_quick_links.mhtml

```diff
@@ -3,14 +3,14 @@
 </%args>
 
 % if (!$read_only_curs) {
-<div class="curs-box">
```

Comment:
I'm not sure about this change.  Doesn't it mean that you can't make a "Genetic interaction" annotation unless you're on advanced mode?

Reply:
You're right; I misread line 13 as referring to only physical interaction annotations, rather than all annotations in the interaction category:

```html
<li ng-if="getAnnotationMode() == 'advanced' || '<% $annotation_type->{category} %>' == 'interaction'">
```
I didn't notice any problems at the time because PHI-Canto doesn't have genetic interaction enabled. I realise now that genetic interaction only applies to metagenotypes, so there isn't really any other place for it at the moment. I guess the only other place would be to add a 'Annotate metagenotypes' heading under 'Annotate genotypes' and put it there, along with the Metagenotype Management link (since that isn't strictly about annotating genotypes either.

For now, can we change this check to apply only to genetic interactions?

Reply:
> For now, can we change this check to apply only to genetic interactions?

I've done that tweak.